app/src/components/ui/organisms/Pomodoro/components.tsx

```diff
@@ -0,0 +1,35 @@
+import { useStore } from '@nanostores/react'
+
+import { handlePhase } from '@store/Pomodoro/actions'
+import { pomodoroStore } from '@store/store'
+import type { PhaseType } from '@store/types'
+
+import Button from '@atoms/Buttons/Button'
+
+import { BUTTON_TEXT } from './constants'
+import './styles.css'
+
+export const PhaseButtons = () => {
+  const { phase } = useStore(pomodoroStore.state)
+
+  const buttonPhases = ['pomodoro', 'break', 'longBreak'] as const
+
+  return (
+    <div className="pomodoro__buttons">
+      {buttonPhases.map((buttonPhase, index) => {
+        let styles = 'button '
```

Comment:
This will always get overwritten, so `button` class will never be used.

You can use a ternary inside a template string so you can set the `styles` final value in 1 line.